### DISCUSSION TITLE
Scrum prep Milestone 3 - Week 1

### DISCUSSION THREAD

**Discussion by keisanabria** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#147] [Scrum Preps]
Scrum prep Milestone 3 - Week 1 (Oct 28-Nov 1)

**Comment by keisanabria** [comment]
Haven't made any progress.

**Comment by Gian-Go** [comment]
I have been busy working with other projects. No work done this week.

**Comment by cristianMarcial** [comment]
haven't done anything, besides defining a team on discussion so that i can have a team to work with

**Comment by Yamil-Serrano** [comment]
I haven't done anything yet, I've just been getting oriented for the milestone 3 ideas.

**Comment by christianmedina-dev** [comment]
Organizing on new teams, joined Registration Team. Also had some work from other courses.

**Comment by Carloscaban** [comment]
Entered registration team, started assigning tasks

**Comment by carlospepin23** [comment]
No work done this week.
